pkg/component/nodemanagement/machinecontrollermanager/machine_controller_manager.go

```diff
@@ -450,7 +551,9 @@ func (m *machineControllerManager) Destroy(ctx context.Context) error {
 		m.emptyDeployment(),
 		m.newShootAccessSecret().Secret,
 		m.emptyService(),
-		m.emptyClusterRoleBindingRuntime(),
+		m.emptyClusterRoleBindingRuntime(), //TODO: @aaronfern Remove this after g/g:v1.119 is released
```

Comment:
```suggestion
		m.emptyClusterRoleBindingRuntime(),
```

Nit: When applying the TODO statement on the function definition, you'll already get pointed to this usage which has to be removed.